api/src/main/java/io/opentelemetry/trace/TracingContextUtils.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace;
+
+import io.grpc.Context;
+import io.opentelemetry.context.ContextUtils;
+import io.opentelemetry.context.Scope;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * Util methods/functionality to interact with the {@link io.grpc.Context}.
+ *
+ * @since 0.1.0
+ */
+@Immutable
+public final class TracingContextUtils {
+  private static final Context.Key<Span> CONTEXT_SPAN_KEY =
+      Context.<Span>key("opentelemetry-trace-span-key");
+
+  /**
+   * Creates a new {@code Context} with the given {@link Span} set.
+   *
+   * @param span the value to be set.
+   * @return a new context with the given value set.
+   * @since 0.1.0
+   */
+  public static Context withSpan(Span span) {
+    return withSpan(span, Context.current());
+  }
+
```

Comment:
Remove this as we did in the CorrelationContext.

Reply:
We still have `Context withCorrelationContext(Correlation Context)` there. Maybe you are talking about removing the one of the overloads of these methods?

Reply:
Yes, let's keep only the version that accepts both the object (span, correlation) and the Context.

Reply:
Done. Lets definitely add them again if/as needed.